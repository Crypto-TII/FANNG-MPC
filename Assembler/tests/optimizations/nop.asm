jmp 0
stms s0, 8201 # 4
jmp 0
stms s0, 8210 # 6
