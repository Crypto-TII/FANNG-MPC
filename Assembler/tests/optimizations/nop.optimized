stms s0, 8201 # 4
stms s0, 8210 # 6
