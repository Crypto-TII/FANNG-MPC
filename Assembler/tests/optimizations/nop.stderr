error: read from uninitialized register
 --> tests/optimizations/nop.asm:2:1
  |
2 | stms s0, 8201 # 4
  | ^^^^^^^^^^^^^^^^^ register 0 was never written to
  |
 ::: tests/optimizations/nop.asm:4:1
  |
4 | stms s0, 8210 # 6
  | ^^^^^^^^^^^^^^^^^ register 0 was never written to
  |